### PR TITLE
Add a functional mvp slide in nav

### PR DESCRIPTION
**Pull request by JulianaTennett**
Hover effect on links, appropriate color changes of hamburger menu, link colors/hovers, and drop shadow

### CODE REVIEW

File: _scripts/main.js [hunk 1]
```diff
@@ -1,4 +1,38 @@
-$( document ).ready( function() {
+import Nav from './modules/nav';
 
-});
+$(document).ready(function() {
+  new Nav();
+  hamburgerColor();
+}); //this creates a jquery object on document then calling .ready() on that jquery object, which takes in a function tnats called afer the jquery obkect is loaded
 
+function checkPage() {
```
Comment (beahuang) [inline _scripts/main.js]: this function isn't being used

File: _scripts/main.js [hunk 2]
```diff
@@ -1,4 +1,38 @@
...
+  var currentPage = window.location.pathname;
+  return currentPage;
+}
+
+function hamburgerColor() {
```
Comment (beahuang) [inline _scripts/main.js]: move this function into Nav.js and call it in the init() function

File: _scss/util/_variables.scss [hunk 3]
```diff
@@ -9,9 +9,13 @@
 $c-black: #4A4A4A;
 $c-white: #fff;
 $c-zealery--pink: #FF685A;
+$c-zealery--pinkshadow: rgba(172, 45, 45, 50);
 $c-zealery--light: #FCF3F2;
+$c-zealery--lightshadow: rgba(106, 17, 17, 50);
 $c-trailtag--teal: #106469;
+$c-trailtag--shadow: rgba(5, 68, 72, 50);
 $c-trailtag--light: #E1E8E8;
+// $c-trailtag--lightshadow: rgba(5, 68, 72, 50);
```
Comment (beahuang) [inline _scss/util/_variables.scss]: delete commented code

File: gulpfile.js [hunk 4]
```diff
@@ -11,5 +11,20 @@ const sync    = require('./gulp/browsersync');
   task( gulp );
 });
 
+gulp.task('icons', function () {
```
Comment (beahuang) [inline gulpfile.js]: where are you using this task?

File: img/icons/hamburger.svg [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (beahuang) [inline img/icons/hamburger.svg]: run this through https://jakearchibald.github.io/svgomg/

File: img/icons/x.svg [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (beahuang) [inline img/icons/x.svg]: https://jakearchibald.github.io/svgomg/

File: gulpfile.js (hunk 4 above)
Comment (JulianaTennett) [reply gulpfile.js]: not, deleted

**Pull request merged**